vic/vic_run/src/calc_atmos_energy_bal.c

```diff
@@ -284,19 +281,18 @@ error_print_atmos_moist_bal(double  VPcanopy,
     ErrorString = (char *)  va_arg(ap, char *);
 
     // print variable values
-    fprintf(LOG_DEST, "%s", ErrorString);
-    fprintf(LOG_DEST, "VPcanopy = %f\n", VPcanopy);
-    fprintf(LOG_DEST, "InLatent = %f\n", InLatent);
-    fprintf(LOG_DEST, "Lv = %f\n", Lv);
-    fprintf(LOG_DEST, "Ra = %f\n", Ra);
-    fprintf(LOG_DEST, "atmos_density = %f\n", atmos_density);
-    fprintf(LOG_DEST, "gamma = %f\n", gamma);
-    fprintf(LOG_DEST, "vp = %f\n", vp);
-    fprintf(LOG_DEST, "AtmosLatent = %f\n", *AtmosLatent);
-
-    log_err("Finished writing calc_atmos_moist_bal variables.\nTry increasing "
-            "CANOPY_VP to get model to complete cell.\nThen check output for "
-            "instabilities.");
+    log_err("VPcanopy = %f\n"
+             "InLatent = %f\n"
+             "Lv = %f\n",
+             "Ra = %f\n",
```

Comment:
Nothing - they should not be there - leftover from previous implementation. Fixed